importer/SeBbrSv.py

```diff
@@ -50,7 +50,8 @@ def set_heritage_bbr(self):
         """
         protection_date = False
         url = "http://kulturarvsdata.se/" + \
-            self.wd_item["statements"][self.props["cultural_heritage_sweden"]][0]["value"]
+            self.wd_item["statements"][
```

Comment:
_I cannot comment on the bits below inline so here goes:_

* I would also add an example of what ns5:spec might look like to explain the string splitting logic.
* On line 70 below. Use a normal comment rather than a multiline string.
* What happens if `utils.date_to_dict` cannot interpret the string as a date?, a try/except would be nice here.